hack/install-deps.sh

```diff
@@ -29,25 +29,37 @@ function cmd() {
 # so we can amend our path to look into the local node_modules for the
 # correct binaries.
 repo_root="$( dirname "${BASH_SOURCE}" )/.."
-export PATH="${PATH}:${repo_root}/node_modules/bower/bin:${repo_root}/node_modules/grunt-cli/bin"
+export PATH="${PATH}:${repo_root}/node_modules/grunt-cli/bin"
 
-# Install bower if needed
-if ! which bower > /dev/null 2>&1 ; then
-  cmd "npm install bower"
+# Install yarn if needed
+if ! which yarn > /dev/null 2>&1 ; then
```

Comment:
Quick Q, does this possibly change a user's existing environment, for example if `yarn` was installed via `brew install yarn`?

Reply:
I may be misunderstanding what you mean, but this block should have no effect as long as the yarn bin is somewhere in the users PATH already (regardless of how it was installed). Using the downloaded install script was the most platform agnostic installation solution I could find.

Reply:
Ah, true, no thats my mistake in reading it.  Should be good!